### ISSUE TITLE
Add GitHub Actions for Testing This Repo

### ISSUE THREAD

**Issue by kevinfodness** [alleyinteractive/create-wordpress-project#15]
# User Story

> As a maintainer of this repo, I want to ensure that changes to this repo don't break automated tests when a full installation, including a plugin and theme, are performed by a user.

# Acceptance Criteria

- Create a GitHub Action to ensure that changes to [create-wordpress-plugin](https://github.com/alleyinteractive/create-wordpress-plugin) and [create-wordpress-theme](https://github.com/alleyinteractive/create-wordpress-theme) do not break this project.
- Ensure these checks run on PRs against this repo and on a schedule.
- Use the automatic installation script established in #7 to ensure that the project is properly configured before testing, emulating user behavior.
- Test against a VIP-type repo and a Pantheon-type repo to ensure both targets are supported.

**Comment by benpbolton** [comment]
### **TLDR;** 

> The AC as listed would be challenging to accomplish with all the mandatory user input prompts. If they were reduced to 'test the defaults' we should be able to get 90% of the way there... if there were --hosting=vip or --hosting=pantheon options, that would make it possible (assuming the rest of the defaults were possible).

@kevinfodness I'm uncertain whether this is actionable yet with the current state of the `configure.php` script (@srtfisher may be able to validate my brief examination).

In particular, when specifying the three command-line options, the user is still prompted (though the defaults are updated). So running, for instance:

```
php ./configure.php --project_name=cwp --author_name=cwp --author_email=cwp@alley.com
```

will still prompt the user: 

![Image](https://github.com/user-attachments/assets/0f6f6b84-53d0-45b6-b54c-90546056efc9)


If we remove the fourth AC (test both Pantheon and VIP), we **can** make some progress here. By leveraging `yes` or another tool or utility, we could test the 'all the defaults' option, which would be better than nothing. For instance:

![Image](https://github.com/user-attachments/assets/f0d11deb-534a-435e-a9d9-21ccf29974fd)


(just accept all defaults). Notably, this seems like it would use the Pantheon hosting option as the default:

![Image](https://github.com/user-attachments/assets/cb4f5852-acd8-471b-ab69-3a9fa4ca1203)


... alternatively a fragile implementation (would break or act unexpectedly when new options or questions are added or removed) we could supply the responses ahead of time (answer yes, yes, no , no, etc)
